### PR TITLE
Add namespace to EventLoop

### PR DESCRIPTION
**Pull request by SubiAchankunju**
The classes and methods of EventLoop library are moved under namespace "eventloop"

### CODE REVIEW

**Comment by anderewrey** [comment]
Sorry to jump in the topic, but I would propose a different way to achieve the same need (and avoiding to end with `eventloop::EventLoop::RegisterEvent(...)`):

Instead of wrapping both `Event` and `EventLoop` classes into a namespace, this can also be done:
1) Changing `class EventLoop` for `namespace EventLoop`
2) Moving the `Event` class into that new namespace.

Indeed, it would need to rewrite a little eventloop.cpp, but since all of its functions and variables are already static, having them into its own namespace makes more sense than a class.

File: include/EventLoop.h [hunk 1]
```diff
@@ -15,11 +15,10 @@
 #include <functional>
 #include <string>
 
-class Event;
+namespace EventLoop {
+
+    class Event;
```
Comment (amoldhamale1105) [inline include/EventLoop.h]: remove indentation, not required in namespace scope

File: include/EventLoop.h [hunk 2]
```diff
@@ -94,5 +93,6 @@ class EventLoop
      * however care must be taken to typecast it correctly at the receiver end before dereferencing to prevent incorrect interpretation or 
      * segmentation faults in case of invalid or NULL pointers
      */
-    static void TriggerEvent(const std::string& evtName, const size_t& timeoutMS, void* data = nullptr);
-};
+    void TriggerEvent(const std::string& evtName, const size_t& timeoutMS, void* data = nullptr);
+
+}//namespace EventLoop
```
Comment (amoldhamale1105) [inline include/EventLoop.h]: leave a space after closing brace and // to make it look a bit cleaner. Make this change wherever applicable 
`} // namespace EventLoop`

**Comment by SubiAchankunju** [comment]
> Sorry to jump in the topic, but I would propose a different way to achieve the same need (and avoiding to end with `eventloop::EventLoop::RegisterEvent(...)`):
> 
> Instead of wrapping both `Event` and `EventLoop` classes into a namespace, this can also be done:
> 
> 1. Changing `class EventLoop` for `namespace EventLoop`
> 2. Moving the `Event` class into that new namespace.
> 
> Indeed, it would need to rewrite a little eventloop.cpp, but since all of its functions and variables are already static, having them into its own namespace makes more sense than a class.

Thanks for the proposal. That point was helpful! The code had been updated based on the suggestion

**Review by amoldhamale1105 — APPROVED** [review]
Looks good, tested with [Shopping cart](https://github.com/amoldhamale1105/ShoppingCart/pull/1) and works as expected. Thank you!

**Pull request merged**